### PR TITLE
added who is oncall related commands

### PR DESCRIPTION
**Pull request by vamshikanduri**

### CODE REVIEW

File: command/schedule_cmd.go [hunk 1]
```diff
@@ -642,6 +643,67 @@ func DeleteScheduleOverrideAction(c *gcli.Context) {
...
+	}
+	if atTime, ok := getVal("atTime", c); ok {
+		t, err := time.Parse(time.RFC3339, atTime)
+		exitOnErr(err)
+		req.Date = &t
+	}
+	flat := c.IsSet("flat")
+	req.Flat = &flat
+
+	resp, err := cli.GetOnCalls(context.Background(), req)
```
Comment (crodjer) [inline command/schedule_cmd.go:664]: Curious, what does `context.Background` do? I don't see it being used by other `cmd`s.

File: lamp.go [hunk 2]
```diff
@@ -2308,6 +2308,94 @@ func deleteScheduleOverrideCommand() gcli.Command {
...
+		gcli.StringFlag{
+			Name:  "name",
+			Usage: "Name of the schedule",
+		},
+		gcli.BoolFlag{
+			Name: "flat, f",
+			Usage: "If the response should be flat",
+		},
+		gcli.StringFlag{
+			Name: "atTime",
```
Comment (crodjer) [inline lamp.go:2326]: Should this be `date`? I agree `atTime` is more descriptive, but we use a different field in the API: https://docs.opsgenie.com/docs/who-is-on-call-api#get-on-calls

File: command/schedule_cmd.go (hunk 1 above)
Comment (vamshikanduri) [reply command/schedule_cmd.go:664]: It would be better to send a default context than nil

File: lamp.go (hunk 2 above)
Comment (vamshikanduri) [reply lamp.go:2326]: yeh, just wanted it to be more friendly to understand

**Review by cglrkn — COMMENTED** [review]
Should we add something to vendor?

**Review by cglrkn — APPROVED** [review]

**Pull request merged**